Source/IntegrationTests/IntegrationTests.csproj

```diff
@@ -0,0 +1,45 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <TargetFramework>net5.0</TargetFramework>
+        <GenerateEmbeddedFilesManifest>true</GenerateEmbeddedFilesManifest>
+        <IsPackable>false</IsPackable>
+        <Nullable>enable</Nullable>
+        <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
```

Comment:
Please remove this for now since it collides with the 'singletons' test run that regretfully currently runs StyleCop because it accidentally picks that up from Boogie.

As long a local `dotnet build Dafny/Source.sln` builds without warnings, I think we're good to merge this PR.